dotnet/sample/AutoGen.BasicSamples/Example10_GenericAPI.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Example08_LMStudio.cs
+
+#region lmstudio_using_statements
```

Comment:
These macros (#region blablabla) are used by website generator. This example probably doesn't need those.